apps/omg_watcher/test/omg_watcher/integration/in_flight_exit_test.exs

```diff
@@ -51,6 +51,11 @@ defmodule OMG.Watcher.Integration.InFlightExitTest do
         [{alice, 5}, {bob, 15}]
       )
 
+    in_flight_tx =
```

Comment:
Please add comment to the `compose_deposit_exit` about the consequences, so you can't compose exit from spent deposit. Also add comment here that we need to prepare these txs before utxos get spend (and refer to the function comment)